cmd/kpod/images.go

```diff
@@ -212,13 +212,21 @@ func getImagesTemplateOutput(runtime *libpod.Runtime, images []*storage.Image, o
 			createdTime = info.Created
 		}
 
+		// units.HumanSizeWithPrecision returns e.g 4.23MB, but want 4.23 MB
+		humanSize := units.HumanSizeWithPrecision(float64(size), 3)
+		if strings.Contains(humanSize, ".") {
+			humanSize = humanSize[:4] + " " + humanSize[4:]
```

Comment:
This seems a little risky.  Why are you putting this space in there?

Reply:
Because the new docker/go-units version prints out the size as `4.23MB`, while we want `4.23 MB` it just looks bad when printed and doesn't match docker images.

Reply:
Well that means we won't match the output of future docker packages.

Also is it guaranteed that :4 and 4: will work for all sizes?

Reply:
BTW Did you take a look at go-units to see why the change was made?

Reply:
We always want a precision of 3 numbers, so if it is of format 4.23, then :4 and 4: will work, but if there is no decimal it will be :3 and 3:.
Yes I did, but couldn't find why such a change was made, they only added a function that lets you specify the precision and returns a string of the digit and unit, but without a space between them.
What do you suggest? Let the size be printed without a space between the value and unit?

Reply:
wait nvm they remove the space to be closer to linux outputs, https://github.com/docker/go-units/commit/9e638d38cf6977a37a8ea0078f3ee75a7cdb2dd1
What should I do? docker images still has the space

Reply:
```
commit 9e638d38cf6977a37a8ea0078f3ee75a7cdb2dd1
Author: Werner Beroux <werner@beroux.com>
Date:   Mon Jan 23 19:20:05 2017 +0100

    Removes spaces before unit.
    
    This is to be closer to common Linux outputs.
    Most Linux tools output with space before the human size unit.
    Examples:
    
     - `du -h`
     - `df -h`
     - `ls -lh`
     - `tree -h`
     - `free -h`
    
    Having a space breaks `sort -h` (and makes things like `awk {print $2}`
    harder).
    
    See https://github.com/docker/docker/issues/30271
```

Reply:
Lets drop this code and just print without the space.

Reply:
ok, fixed

Reply:
We can fix the tool that checks the output to ignore the space.